test/e2e/libpod_suite_test.go

```diff
@@ -148,7 +150,17 @@ func PodmanCreate(tempDir string) PodmanTest {
 		cgroupManager = os.Getenv("CGROUP_MANAGER")
 	}
 
+	// Ubuntu doesn't use systemd cgroups
+	if host.Distribution == "ubuntu" {
+		cgroupManager = "cgroupfs"
+	}
+
 	runCBinary := "/usr/bin/runc"
+	altRunCBinary := "/usr/sbin/runc" // ubuntu
+	if _, err := os.Stat(runCBinary); err != nil {
```

Comment:
could we use `os.LookPath("runc")` here?

Reply:
given that /usr/sbin should be in the path, I guess it would work nicely ... everyone prefer that?

Reply:
fixed